OpenTrons/opentrons-api
# feat(app): add privacy policy acknowledgement screen on ODD

closes [RAUT-818](https://opentrons.atlassian.net/browse/RAUT-818)

# Overview

Adds a privacy policy acknowledgment ODD component to a that is starting up the robot for the first
time after upgrading to 7.2. Will only be shown if the reading for `getAnalyticsOptInSeen` is null,
meaning the user has not yet read and acknowledged the new privacy policy. Once accepted, the value
will be set to `true`, and the user will not be prompted with the policy screen again.

# Test Plan

Recreating unboxing on 7.2.0:
- ssh into robot and open data/ODD/config.json
- set analytics values for `optedIn` and `seenOptedIn` to `false`
- set `onDeviceDisplaySettings: unfinishedUnboxingFlowRoute` to `"/welcome"
- restart ODD (from top level ssh, `systemctl restat opentrons-robot-app`)
- observe analytics opt in during unboxing flow

Recreating unboxed robot after upgrade to 7.2.0:
- ssh into robot and open data/ODD/config.json
- set analytics values for `optedIn` and `seenOptedIn` to `false`
- set `onDeviceDisplaySettings: unfinishedUnboxingFlowRoute` to `null`
- restart ODD (from top level ssh, `systemctl restat opentrons-robot-app`)
- observe analytics opt-in page at robot dashboard on reboot

# Changelog

- create new `PrivacyPolicy` ODD page
- conditionally route to `PrivacyPolicy` page from `EmergencyStop` and `RobotDashboard` depending on analytics config values not both being `true`
- modify welcome modal close function to modify ODD config for `unfinishedUnboxingFlowRoute` to not launch welcome modal again
- remove deprecated analytics opt-in modal
- add new translations

# Risk assessment

medium

[RAUT-818]: https://opentrons.atlassian.net/browse/RAUT-818?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
